.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] Kindly add `test.md` file to `.gitignore` to not push the file to the remote repository when you push changes 👍